src/devoxx/model.rs

```diff
@@ -37,11 +36,23 @@ pub struct ScheduleItem {
     pub room_name: String,
     pub from_date: DateTime<Utc>,
     pub to_date: DateTime<Utc>,
-    pub session_type_name: Option<String>
+    pub session_type_name: Option<String>,
+    pub speakers: Option<Vec<Speaker>>,
```

Comment:
Would an empty vec would suffice instead of an optional here ?

Reply:
`speakers` can be null in the json, which gave a parsing error when it was not wrapped in an Option.

Could implement a deserializer function for that field (and possibly others): https://stackoverflow.com/a/44210829/6797663